src/com/lftechnology/java/training/alina/jdbc/service/UserService.java

```diff
@@ -0,0 +1,181 @@
+package com.lftechnology.java.training.alina.jdbc.service;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import com.lftechnology.java.training.alina.jdbc.constants.Constants;
+import com.lftechnology.java.training.alina.jdbc.controller.EmployeeController;
+import com.lftechnology.java.training.alina.jdbc.controller.LoginController;
+import com.lftechnology.java.training.alina.jdbc.dao.user.impl.UserDaoImpl;
+import com.lftechnology.java.training.alina.jdbc.domain.Database;
+import com.lftechnology.java.training.alina.jdbc.domain.Employee;
+import com.lftechnology.java.training.alina.jdbc.domain.User;
+import com.lftechnology.java.training.alina.jdbc.employeeenum.EmployeeRole;
+/**
+ * UserService consists of all functionalities of user
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class UserService {
+
+    private static final Logger LOGGER = Logger.getLogger(UserService.class.getName());
+
+    /**
+     * Checks for user login
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param user
+     *            {@link User}
+     * @return isLogin {@link Boolean} true if login success, false if login fails
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public Boolean checkEmployeeLogin(Scanner scanner, User user) throws SQLException {
+        UserDaoImpl userDao = new UserDaoImpl();
+        boolean isLogin = userDao.checkEmployeeLogin(scanner, user);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "isLogin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)